website/src/content/api-reference/commercetools-frontend-application-shell-connectors.mdx

```diff
@@ -72,6 +72,42 @@ The information about the logged in user.
 }
 ```
 
+For users who have logged-in into the Merchant Center using SSO, there will be another object property (**idTokenUserInfo**) which will contain some standard claims from the id token we get in our backend when processing the OpenId Connect (OIDC) flow with their Identify Provider.
```

Comment:
```suggestion
If the user logged into the Merchant Center using [Single Sign-On](https://docs.commercetools.com/merchant-center/single-sign-on-beta), we expose an additional property `idTokenUserInfo` which contains some of the user info claims (OpenID Connect) from the user's Identity Provider.
```

Reply:
Thanks for the suggestion.

Updated in this commit: 6898054f37f9176a6f1fadd0c0766f4d64454c6b